libbeat/docs/processors-using.asciidoc

```diff
@@ -748,3 +749,33 @@ The fields added to the event are looking as following:
 -------------------------------------------------------------------------------
 
 NOTE: The host information is refreshed every 5 minutes.
+
+[[dissect]]
+=== Dissect
+
+The `dissect` processor allows you to use the `dissect` syntax to tokenizer incoming string into
+defined patterns.
+
+
+Note that if the key exists in the `target_field`, the original value will be replaced.
+
+TODO: need to finish
```

Comment:
@dedemorton  Just to keep you in the loop, since both beat and LS will share the same syntax and behavior, should we try to make a high definition/examples of the syntax and share it with between the two project or not?

Reply:
@ph I think that's a good idea, but there are a few things to consider:

* Where will the shared asciidoc source files live? Will there be a new repo for code/docs shared across ingest? 
* Will the behavior across LS and Beats always be identical? Will the projects share code? If we add a new processor to Beats, will we always add it to Logstash in the same release (and vice versa).

Reply:
> Where will the shared asciidoc source files live? Will there be a new repo for code/docs shared across ingest?

Good question, I think for now we can duplicate the content but have the same description.

> Will the behavior across LS and Beats always be identical? Will the projects share code? If we add a new processor to Beats, will we always add it to Logstash in the same release (and vice versa).

For now we only have dissect and in the short term we are not planning anything else.

Reply:
For now i would keep it separate to make sure it does not hold back the progress of this PR. We can still do a follow up PR / discussion no how to merge it.

Reply:
@ruflin I will make the docs PR as a separate PR.

Reply:
@DanielYWoo IIRC Dissect appeared in LS two years ago.